modules/cnf-configuring-kubelet-nro.adoc

```diff
@@ -41,7 +41,7 @@ spec:
       memory: "512Mi"
     topologyManagerPolicy: "single-numa-node" <5>
 ----
-<1> Adjust this label to match the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR.
+<1> Ensure this label matches the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR configured later in "Creating the NUMAResourcesOperator custom resource".
```

Comment:
On HyperShift, the selector has no actual meaning, the association to the nodes is done via the desired `nodePool`